nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/sql/S3DirectBatchHandler.java

```diff
@@ -75,13 +74,11 @@
 
     public static final String POLICY_TEMPLATE_PROPERTY = "policyTemplate";
 
-    protected static final List<String> MANDATORY_PROPERTIES = Arrays.asList( //
-            AWS_ID_PROPERTY, //
-            AWS_SECRET_PROPERTY, //
-            BUCKET_NAME_PROPERTY, //
-            BUCKET_REGION_PROPERTY, //
-            POLICY_TEMPLATE_PROPERTY //
-    );
+    protected static final List<String> MANDATORY_PROPERTIES = Arrays.asList(
+            AWS_ID_PROPERTY,
```

Comment:
Please keep the `//` on each line, it's necessary for some IDEs (Eclipse) to not reformat everything on one line.